src/fontra/views/editor/editor.js

```diff
@@ -933,8 +941,9 @@ export class EditorController {
       return;
     }
 
-    const preferGLIF = true; // TODO should be user preference
-
+    const preferGLIF =
```

Comment:
Given the multi-state options UI, this should also reflect that. We should probably use a string, and name the variable "clipboardExportFormat". Values: "svg", "glif", "fontra-json". Perhaps the values could be literally the value in localStorage.